### ISSUE TITLE
NTR: Abnormal urine D-xylose test (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8876]
New term request Abnormal urine D-xylose test (nadea3_200205152321): D-xylose absorption test abnormal  UMLS: 0580568

**Comment by pnrobinson** [comment]
@MickeySegal
the intention of this term
[Small intestinal bacterial overgrowth HP:0033163](https://hpo.jax.org/browse/term/HP:0033163)
is to represent an abnormal result of this and similar tests. 
Perhaps it is sufficient to add this test result as a related synonym?
Is the test diagnostic of any other condition?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001082
- label: Abnormal urine D-xylose test
- parents: [Functional abnormality of the gastrointestinal tract (HP:0012719)](https://hpo.jax.org/app/browse/term/HP:0012719)
- pmid: PMID:11418788
- def: An abnormal result of the xylose absorption test that is used to assess the ability of the intestines to absorb D-xylose. D-xylose is a five-carbon monosaccharide found in some plants. The patient drinks a solution containing a specific amount of D-xylose and the concentration of xylose is measured in the blood circulation and then in the urine. A low amount of D-xylose indicates reduced small intestinal absorbtion.
- comment: The standard 25-g D-xylose test in adults, based on analysis of a 5-h urine collection and a 1-h serum sample, discriminates between normal subjects and patients with proximal small intestinal malabsorption with X5% specificity and sensitivity.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8876
- synonyms:
